test/e2e/config/conf.js

```diff
@@ -36,13 +47,21 @@ const config = {
   framework: 'custom',
   frameworkPath: require.resolve('protractor-cucumber-framework'),
   specs: ['../features/**/*.feature'],
-  baseUrl: process.env.TEST_URL || 'http://localhost:2500',
+  // specs: [
+  //   '../features/**/caseFile.feature',
+  //   '../features/**/login.feature',
+  //   '../features/**/makeDecision.feature',
+  //   '../features/**/parties.feature',
+  //   '../features/**/questions.feature',
+  //   '../features/**/recentEvents.feature',
+  // ],
+  baseUrl: process.env.TEST_URL || 'http://localhost:3000',
```

Comment:
please add this to config file (http:localhost.... )

Reply:
feature improvements